nightfall-optimist/src/routes/block.mjs

```diff
@@ -16,6 +13,133 @@ import { setMakeNow } from '../services/block-assembler.mjs';
 
 const router = express.Router();
 
+/**
```

Comment:
Hey friends, I'm not sure if this is the best way, but having such big block of comments inside the source-code doesn't look good to me. Maybe, moving them to a YML file apart from the source-code it is better. Having this separated might be easier to maintain. We should discuss that with the team, maybe I'm wrong and this is the right of of doing it for JS apps.

Reply:
Like we talked yesterday, they implemented these docs first on a separate file - like you suggest, and people seemed to complain. I don't think we're happy with the result... Let's decide later. Since we're working on a feature branch we have some time to reflect/research/decide.
:pray: